velox/exec/HashAggregation.h

```diff
@@ -72,6 +72,16 @@ class HashAggregation : public Operator {
 
   RowVectorPtr getDistinctOutput();
 
+  // Setup the projections for accessing grouping keys stored in the grouping
+  // set. 'groupingKeyInputChannels' provides the map from the key column index
+  // in the grouping set (as vector index) to key column channel in the input
+  // (as vector value). 'outputChannelProjections' provides the map from the key
+  // column channel in the output (as vector index)  to the key column index in
+  // the grouping set (as vector value).
+  void setupGroupingKeyChannelProjections(
```

Comment:
s/setupGroupingKeyChannelProjections/createGroupingKeyChannelProjections/
since we are returning the projections instead of setting it up for the containing object

Reply:
I think it is better to call setup for projections